scripts/circle-ci/trigger-antithesis-simulation.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ -z "${ANTITHESIS_WEBHOOK_PASSWORD}" ]]; then
+  echo "Antithesis webhook password is not set as an environment variable, exiting."
+  exit 1
+fi
+
+WEBHOOK_LOCATOR="palantir_atlasdb__smoketest__latest"
```

Comment:
This is not the final webhook we'll be publishing to, but using the smoke test one for now to verify everything works without having to spend too much of our quota by triggering real experiments. 

If everything in this PR works as we expect, I'll update the weblocator and frequency of run in a next PR.